webpack.config.dev.js

```diff
@@ -5,7 +5,7 @@ exports.entry.unshift('webpack/hot/dev-server');
 exports.module.loaders[0].loaders = ['babel', 'react-hot', 'jsx?harmony'];
 exports.plugins[1] = new webpack.DefinePlugin({
   __PROD__: false,
-  __S3_BUCKET__: 'landline'
+  __S3_BUCKET__: JSON.stringify('landline')
```

Comment:
It might make sense at some point to move the api url from the AppStore to this object, since it never has to change.